AirCasting/SearchAndFollow/SearchMap/SearchMapViewModel.swift

```diff
@@ -41,9 +42,18 @@ class SearchMapViewModel: ObservableObject {
         self.sensorType = sensorType
     }
     
+    func textFieldTapped() { isLocationPopupPresented.toggle() }
     func getMeasurementName() -> String { measurementType.capitalizedName }
     func getSensorName() -> String { sensorType.capitalizedName }
     
+    func locationNameInteracted(with newLocationName: String) {
```

Comment:
what do you think about `newLocationEntered(name: String)` / `enteredNewLocation(name: String)` ?